include/nuttx/compiler.h

```diff
@@ -61,6 +61,14 @@
 #  define CONFIG_C99_BOOL 1
 #endif
 
+#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201112L
+#  define CONFIG_C11_BOOL 1
+#endif
+
+#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201710L
+#  define CONFIG_C17_BOOL 1
```

Comment:
Where is it used?

Reply:
This is for future compatibility, in conjunction with the other ones

Reply:
it's better to create a new patch and PR instead